src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx

```diff
@@ -16,80 +16,83 @@ import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application
+                operator.name = response;
+                applicationStore.setIsLoading(false);
+                const selectedOperator = myAccountStore.ownerAddressOperators.find((op: any) => op.id === operator.id);
+                selectedOperator.name = response;
+                console.log(response);
+                navigate(-1);
+            });
+        } catch (error: any) {
+            console.log('<<<<<<<<<<<error>>>>>>>>>>>');
+            setErrorMessage(error.message);
+            console.log('<<<<<<<<<<<error>>>>>>>>>>>');
+            applicationStore.setIsLoading(false);
+        }
+    };
 
-  const setOperatorName = async (e: any) => {
-    const textInput = e.target.value.trim();
-    setUserInput(textInput);
-  };
+    const setOperatorName = async (e: any) => {
+        const textInput = e.target.value.trim();
+        setUserInput(textInput);
+    };
 
-  const submitDisable = userInput.length === 0;
+    const submitDisable = userInput.length === 0;
 
-  return (
-      <BorderScreen
-          blackHeader
-          header={'Change Operator Name'}
-          body={[
-            (
-                <Grid container item>
-                  <Grid container style={{ gap: 24 }}>
-                    <Grid item className={classes.Text}>
-                      {/*Enter an Ethereum address that will receive all of your validators block proposal rewards. <LinkText text={'What are proposal rewards?'} link={'http://google.com'} />*/}
+    return (
+        <BorderScreen
+            blackHeader
+            header={'Change Operator Name'}
+            body={[
+                (
+                    <Grid container item>
+                        <Grid container style={{ gap: 24 }}>
+                            <Grid item className={classes.Text}>
+                                {/*Enter an Ethereum address that will receive all of your validators block proposal rewards. <LinkText text={'What are proposal rewards?'} link={'http://google.com'} />*/}
+                            </Grid>
+                        </Grid>
+                        <Grid container gap={{ gap: 17 }}>
+                            <Grid item container>
+                                <InputLabel title="Operator Name"/>
+                                <TextInput
+                                    value={userInput}
+                                    disable={readOnlyState}
+                                    showError={!isAddressValid}
+                                    data-testid="new-operator-name"
+                                    onChangeCallback={setOperatorName}
+                                    icon={<Grid onClick={() => setReadOnlyState(false)} className={classes.EditIcon}/>}
+                                />
+                                {/*<Grid className={classes.ErrorText}>{!isAddressValid ? 'Invalid address, please input a valid Ethereum wallet address' : ''}</Grid>*/}
```

Comment:
If this should not be used just remove it

Reply:
done